Errors on build and launching the language server

Thanks for the hard work! ❤️

I got the following error while running the command `pnpm run build`, even though the CI passes. 😕 

#### ❓ Error 1

```
> @ build C:\Users\xxx\workspace\grammarly-language-server
> rollup -c && node scripts/build-web-extension.mjs


packages\grammarly-languageclient\src\index.ts → packages/grammarly-languageclient/dist/index.d.ts...
packages/grammarly-languageclient/src/protocol.ts(23,11): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the 'lib' compiler option to include 'dom'.
packages/grammarly-languageclient/src/protocol.ts(26,13): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the 'lib' compiler option to include 'dom'.
packages/grammarly-languageclient/src/protocol.ts(29,13): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the 'lib' compiler option to include 'dom'.

[!] (plugin dts) Error: Failed to compile. Check the logs above.
...
```

but once I have removed the `console.log` in `packages/grammarly-languageclient/src/protocol.ts` then it built correctly!

```ts
...
        return async (...args: unknown[]): Promise<unknown> => {
          //console.log('Request:', property, ...args)
          try {
            const result = await client.sendRequest(`$/${property}`, args)
            //console.log('Response:', property, result)
            return result
          } catch (error) {
            //console.log('Error:', property, error)
            throw error
          }
...
```

then I tried to start the server, another error occurred.

#### ❓ Error 2

```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '../dist/index.cjs'
Require stack:
- C:\Users\xxx\workspace\grammarly-language-server\packages\grammarly-languageserver\bin\server.js
←[90m    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)←[39m
←[90m    at Function.Module._load (node:internal/modules/cjs/loader:778:27)←[39m
←[90m    at Module.require (node:internal/modules/cjs/loader:1005:19)←[39m
←[90m    at require (node:internal/modules/cjs/helpers:102:18)←[39m
    at Object.<anonymous> (C:\Users\xxx\workspace\grammarly-language-server\packages\grammarly-languageserver\bin\server.js:3:16)
←[90m    at Module._compile (node:internal/modules/cjs/loader:1103:14)←[39m
←[90m    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)←[39m
←[90m    at Module.load (node:internal/modules/cjs/loader:981:32)←[39m
←[90m    at Function.Module._load (node:internal/modules/cjs/loader:822:12)←[39m
←[90m    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)←[39m {
  code: ←[32m'MODULE_NOT_FOUND'←[39m,
  requireStack: [
    ←[32m'C:\\Users\\xxx\\workspace\\grammarly-language-server\\packages\\grammarly-languageserver\\bin\\server.js'←[39m
  ]

```

And realized the `dist/index.cjs` does not exists in the language server. 😖

--- a/.changeset/shy-moons-poke.md
+++ b/.changeset/shy-moons-poke.md
@@ -1,0 +1,5 @@
+---
+'grammarly-languageserver': patch
+---
+
+Fix import path in grammarly-languageserver bin

--- a/packages/grammarly-languageclient/src/protocol.ts
+++ b/packages/grammarly-languageclient/src/protocol.ts
@@ -20,13 +20,10 @@ export function createProtocol(client: BaseLanguageClient): Protocol {
         return (fn: (...args: unknown[]) => unknown) => client.onNotification(`$/${property}`, fn)
       } else {
         return async (...args: unknown[]): Promise<unknown> => {
-          console.log('Request:', property, ...args)
           try {
             const result = await client.sendRequest(`$/${property}`, args)
-            console.log('Response:', property, result)
             return result
           } catch (error) {
-            console.log('Error:', property, error)
             throw error
           }
         }

--- a/packages/grammarly-languageserver/bin/server.js
+++ b/packages/grammarly-languageserver/bin/server.js
@@ -1,5 +1,5 @@
 #!/usr/local/env node
 
-const server = require('../dist/index.cjs')
+const server = require('../dist/index.node.cjs')
 
 server.startLanguageServer()

--- a/packages/grammarly-languageserver/package.json
+++ b/packages/grammarly-languageserver/package.json
@@ -77,7 +77,7 @@
   },
   "devDependencies": {
     "domhandler": "^5.0.3",
-    "node-fetch": "^3.2.4"
+    "node-fetch": "^2.0.0"
   },
   "peerDependencies": {
     "node-fetch": "^2.0.0 | ^3.0.0"

--- a/packages/grammarly-languageserver/src/polyfill-fetch.ts
+++ b/packages/grammarly-languageserver/src/polyfill-fetch.ts
@@ -8,11 +8,11 @@ if (typeof global !== 'undefined' && typeof global.fetch === 'undefined') {
     global.Headers = Headers
   })()
 
-  function getNodeFetch(): any {
+  async function getNodeFetch(): any {
     try {
-      return require('node-fetch')
+      return await import('node-fetch')
     } catch {
-      return import('node-fetch')
+      return require('node-fetch')
     }
   }
 }

